<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>debt</class>
 <widget class="QDialog" name="debt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>151</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <widget class="QLabel" name="minimum_lbl">
      <property name="text">
       <string>Minimum</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="principal_line_edit">
      <property name="text">
       <string>40</string>
      </property>
      <property name="placeholderText">
       <string>Principal</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="principal_lbl">
      <property name="text">
       <string>Principal</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="interest_lbl">
      <property name="text">
       <string>Interest</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="interest_line_edit">
      <property name="text">
       <string>.04</string>
      </property>
      <property name="placeholderText">
       <string>Interest</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="minimum_line_edit">
      <property name="text">
       <string>10</string>
      </property>
      <property name="placeholderText">
       <string>Minimum</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="name_lbl">
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="name_line_edit">
      <property name="text">
       <string>credit card</string>
      </property>
      <property name="placeholderText">
       <string>Name</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>160</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="done_btn">
      <property name="text">
       <string>Done</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="add_btn">
      <property name="text">
       <string>Again</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="title_lbl">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Debt</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
